README.md

```diff
@@ -209,19 +201,23 @@ Generally, these would include all the infrastructure, cloud, services, licenses
 The primary ongoing development costs not related to feature development on this platform, in addition to responding to events and the infrastructure itself, will be the updates to the application libraries to address ongoing security concerns. The application frontend is by far the most vulnerable part of this system in terms of security and will require ongoing updates. The subsequent phase, if applicable and if chosen for the semifinals, will provide an expected high-level budget of operational costs and expected expenses in this category.
 
 
-### f. Data Integrity
+### d. Security
 
-The candidate's career path and progress, as well as non-profit information, are accessed by multiple personas. It is crucial to ensure the data is accessed by the authorized person, and the accuracy of the data is maintained throughout its lifecycle.
+The application is exposed on the web which always has the risk of cyberattacks; should the application be accessed by unauthorized users. Moreover, the system stores sensitive information, such as personal data.
+
+### e. Privacy
 
-### g. Responsiveness
+The system needs to handle the PII of candidates. Personal information must be accessed by authorized administrators, mentors, and non-profit representatives.
+
+### f. Interoperability
 
-Since Spotlight is user-facing, it is important to ensure the system responds to the end-user requests promptly. All user interactions *MUST* be coded in such a way as to provide some form of feedback for all actions. This feedback must occur within the [Doherty Threshold of 400ms](https://www.uxtoast.com/ux-laws/doherty-threshold).
+The purpose of this product is to facilitate collaboration between non-profits. To achieve this goal, Spotlight needs to communicate with external systems (such as childcare/housing services) and legacy systems.
 
-### h. Availability
+### g. Data Integrity
 
-Amazon provides services in multiple Availability Zones. By using Serverless services, the Diversity Cyber Council platform will make use of these, by default and as such, the platform can withstand the loss of a given Availability Zone.
+The candidate's career path and progress, as well as non-profit information, are accessed by multiple personas. It is crucial to ensure the data is accessed by the authorized person, and the accuracy of the data is maintained throughout its lifecycle.
 
-### j. Scalability
+### i. Scalability
```

Comment:
So I was thinking... shouldn't our solution be elastic? I don't ever see an increasing amount of load - i see a more burst type load that would happen after work hours. Maybe we can elaborate on this and compare to load patterns at other course sites or job search sites?

Reply:
um I did not remove scalability. My main focus for this MR was on the top 3: Usability, Responsiveness, Feasibility.
Regarding the scalability
I am seeing main flows: non-profit/candidate registration (write to DB) AND non-profit search (mostly read operations).
The first flow is done once when the non-profit/candidate registers, and some steps are asynchronous. I don't expect too much traffic here. Viewing the content (pdfs, rich text) is not going to put too much load on the system if we are using CDN (noticed that we don't consider CDN in our architecture). However querying non-profits might impose load on the system. 
All in all, I agree scalability is important but it is not probably in the top 3. That's my 2 cents :)
PS: we need to spend a few minutes as a team sorting the remaining 5 characteristics. I identified the top 3 (it is open to discussion for sure :D), but the rest are not sorted.

Reply:
Oh, no I know you didn't remove scalability. My comment was basically maybe we should remove it and replace it with elasticity instead. I agree let's chat in a meeting re: these characteristics

Reply:
Note to us, remove the "if selected ..." text

Reply:
For each characteristic that we do choose, we should block-quote the text in the requirements that the characteristic was infered from.